src/middleware/secure-file-storage-api/src/main/java/com/nttdata/securefilestorageapi/controller/FileController.java

```diff
@@ -8,16 +8,17 @@
 import org.springframework.web.multipart.MultipartFile;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 @RestController
-@RequestMapping("/files")
+@CrossOrigin("http://localhost:3085")
```

Comment:
can we make this an env var so each person can set it themselves? for ex, my fe is running on 3000 not 3085

Reply:
I think I got that in?  Let me know if it isn't working...